oshi-core/src/main/java/oshi/hardware/platform/unix/solaris/SolarisCentralProcessor.java

```diff
@@ -52,6 +52,15 @@
     private static final long serialVersionUID = 1L;
 
     private static final Logger LOG = LoggerFactory.getLogger(SolarisCentralProcessor.class);
+    
+    private static final long BOOTTIME;
+    static {
+    	Kstat ksp = KstatUtil.kstatLookup("unix", 0, "system_misc");
+        if (ksp != null && KstatUtil.kstatRead(ksp)) {
+            BOOTTIME = KstatUtil.kstatDataLookupLong(ksp, "boot_time");
+            break;
+        }
```

Comment:
You need to handle the case where the final variable isn't initialized in the conditional.